packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -34,14 +39,18 @@ export const App = ({
               <CannotCreateAccount path="/cannot_create_account/*" />
               <Clear path="/clear/*" />
               <CookiesDisabled path="/cookies_disabled/*" />
-              <ResetPassword path="/reset_password/*" />
-              <ConfirmResetPassword path="/confirm_reset_password/*" />
-              <ResetPasswordWithRecoveryKeyVerified path="/reset_password_with_recovery_key_verified/*" />
+
               <Legal path="/legal/*" />
               <LegalTerms path="/legal/terms/*" />
               <LegalTerms path="/:locale/legal/terms/*" />
               <LegalPrivacy path="/legal/privacy/*" />
               <LegalPrivacy path="/:locale/legal/privacy/*" />
+
+             <ResetPassword path='/reset_password/*' />
+             <ConfirmResetPassword path='/confirm_reset_password/*' />
+             <CompleteResetPassword path='/complete_reset_password/*' />
+             <ResetPasswordConfirmed path='/reset_password_verified/*' />
```

Comment:
Did you also look into `reset_password_confirmed`? Not sure if we need to set up a redirect or anything from it to `/verified`.

Reply:
We could, currently after a password reset the user is redirected back to signin and we have a banner that displays they need to relogin. This might be a view we can remove?

Reply:
Fine by me to remove, it's confusing having the dupe. I'd be surprised if we linked anywhere to this (like SUMO or in-browser somewhere)